packages/flutter/lib/src/scheduler/scheduler.dart

```diff
@@ -0,0 +1,149 @@
+// Copyright 2015 The Chromium Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+import 'dart:async';
+
+import 'package:collection/priority_queue.dart';
+import 'package:flutter/animation.dart' as animation show scheduler;
```

Comment:
Where is this intended to fit in the layering?:
https://docs.google.com/presentation/d/1cw7A4HbvM_Abv320rVgPVGiUP2msVs7tfGbkgdrTy0I/edit#slide=id.gbb3c3233b_0_162